src/components/NavBarComponent2.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
Adding a 2 after the name is not a good naming convention, instead use something like "PrimaryNavbar" and "SecondaryNavbar" if you need more than one and can't just make one customizable component using props.